sotodlib/toast/sim_hwpss.py

```diff
@@ -43,8 +44,19 @@ def exec(self, data):
         for obs in data.obs:
             tod = obs["tod"]
             focalplane = obs["focalplane"]
+            
             # Get HWP angle
             chi = tod.local_hwp_angle()
+            
+            # Get times
+            times = tod.local_times()
+            
+            # Fast sample HWP angle with a factor 100
+            fast_time = np.linspace(times[0], times[-1], 1+(len(times)-1)*100)
+            hwp_rate = (chi[1] - chi[0]) / (times[1] - times[0]) / (2*np.pi) # Assuming constant HWP rotation
```

Comment:
I don't think we can assume constant rotation rate. We know that is not the case in real experiments.

Reply:
This is no longer a requirement in the latest fix.